src/components/missions/Missions.js

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { React, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Mission from './Mission';
+import missionsFetch from '../../redux/missions/missionsFetch';
 
-function Missions() {
-  return <div />;
-}
+const Missions = () => {
+  const missions = useSelector((state) => state.missionsReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => { dispatch(missionsFetch()); }, []);
+
+  return (
+    <div>
+      <table>
+        <tbody>
+          <tr>
+            <th>Missions</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th>Action</th>
+          </tr>
+          {missions.map(
+            (mission) => (
+              <Mission
+                key="mission_id"
```

Comment:
* Using `id` gotten directly from JSON file as a key could cause `id` repetition at some point except they are very unique, as it could make two children have the same `id`. Using third-party id generators instead could help remedy this situation. Could you please consider switching to a third-party `id` generator to make your code more robust? You can click [here](https://www.npmjs.com/package/uuid)  to know more.